radioactive-decay/info.yml

```diff
@@ -0,0 +1,14 @@
+---
+uid: radioactive_decay 
+title: radioactive decay 
+prerequisites:
+  - undergraduate statistics and probability
+learning_objectives:
+  - basic understanding of radioactivity
```

Comment:
It's ok if we combine this content with a more in depth node on decay too. I was just going to take all the content I have and put it up for everyone to use. I do link to other material that covers decay in more detail in the OER.